packages/react-liveness/src/components/FaceLivenessDetector/service/machine/__tests__/index.test.ts

```diff
@@ -530,14 +530,16 @@ describe('Liveness Machine', () => {
       const clientInfo =
         mockLivenessStreamProvider.sendClientInfo.mock.calls[0][0];
 
+      const videoEl = service.state.context.videoAssociatedParams?.videoEl!;
+      Object.defineProperty(videoEl, 'videoHeight', { value: 100 });
       expect(
         expect(
           clientInfo.Challenge.FaceMovementAndLightChallenge.InitialFace
             .BoundingBox
         ).toStrictEqual({
-          Height: 0,
+          Height: -5 / 12,
```

Comment:
Nit: little weird to me to mix fractions and decimal values in the same assertion

Reply:
Lol it's a recurring decimal so I was worried about issues with the precision and StrictEqual but that's fair